bin/si/src/cli.ts

```diff
@@ -181,7 +181,7 @@ function buildCommand() {
     // deno-lint-ignore no-explicit-any
     .command("ai-agent", buildAiAgentCommand() as any)
     // deno-lint-ignore no-explicit-any
-    .command("mcp-server", buildMcpServerCommand() as any)
+    .command("mcp-server", buildMcpServerCommand().hidden() as any)
```

Comment:
Neat! Didn't know about this one

Reply:
yeah this still makes it a valid CLI option :D